experiments/conditions.go

```diff
@@ -49,12 +49,6 @@ func calculateExperimentConditions(experiment *v1alpha1.Experiment, newStatus v1
 		msg := fmt.Sprintf(conditions.ExperimentRunningMessage, experiment.Name)
 		condition := conditions.NewExperimentConditions(v1alpha1.ExperimentProgressing, corev1.ConditionTrue, conditions.NewRSAvailableReason, msg)
 		conditions.SetExperimentCondition(&newStatus, *condition)
-	case conditions.ExperimentTimeOut(experiment, newStatus):
-		// Update the experiments with a timeout condition. If the condition already exists,
-		// we ignore this update.
-		msg := fmt.Sprintf(conditions.ExperimentTimeOutMessage, experiment.Name)
-		condition := conditions.NewExperimentConditions(v1alpha1.ExperimentProgressing, corev1.ConditionFalse, conditions.TimedOutReason, msg)
-		conditions.SetExperimentCondition(&newStatus, *condition)
```

Comment:
I removed TimeOut condition from experiments because this now fails the Experiment, and will be implicit by the Failed status. One less state to manage.